src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetLivePairing/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box } from 'grommet';
```

Comment:
When you open/close the widget, that current state (paring secret && timer bar) of the pariring button is lost.

Reply:
I think it's because the component is unmounted... I don't think the behaviour can be changed without reworking most of the widget behavior. WDYT ?